homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,32 @@
 #pragma once
 
-int NWD(int lhs, int rhs) {
+int NWD(int lhs, int rhs) 
+{
     // TODO: Implement me :)
-    return -1;
+    //OK
+  
+    
+    int kwik;
```

Comment:
Improper name.
Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...